off-chain/src/Network/Types.purs

```diff
@@ -263,26 +267,30 @@ newtype LeaderNode a = LeaderNode
 
 derive instance Newtype (LeaderNode a) _
 
-newtype UserState a = UserState
-  {
-    -- | `actionsSent` pourpose is to store the actions already
-    -- | send the to the `leader`that are confirmed to be accepted
-    -- | but are still waiting to reach the requirement of signature.
-    actionsSent :: Ref (OrderedMap UUID a)
-  -- | For actions whose transaction is already signed and sent 
-  -- | to the server.
-  , transactionsSent :: Ref (OrderedMap UUID a)
-  -- | Those signed transactions are confirmed to be in the chain 
-  -- | ready for submission.
-  -- | Once a transaction is confirmed to be done, we can safely 
-  -- | remove it from this.
-  -- | Is advisable to clean it from time to time, Seath WON'T clean it.
-  -- Well maybe we can put a extra call back in the interface that
-  -- is automatically called wen the transaction is confirmed in the 
-  -- blockchain and then we autoremove this transactions.
-  , submitedTransactions :: Ref (OrderedMap UUID a)
+type UserStateInner a =
+  { actionsSentQueue ::
```

Comment:
As I understood, we are sending status-change events to this queue. Maybe rename it to something that reflects that?

Reply:
Maybe something like `actionsSentEventDistpatcher`?  honestly, I prefer `actionsSentQueue`, but if you got with something interesting, sure.